donut/modules/groups/helpers.py

```diff
@@ -237,10 +261,25 @@ def create_position_holder(pos_id, user_id, start_date, end_date):
         start_date: Starting date of the holding period, format is 'yyyy-mm-dd'
         end_date: end date of the hold period
     '''
-    s = """INSERT INTO position_holders (pos_id, user_id, start_date, 
+    s = """INSERT INTO position_holders (pos_id, user_id, start_date,
     end_date) VALUES (%s, %s, %s, %s)"""
     with flask.g.pymysql_db.cursor() as cursor:
         cursor.execute(s, (pos_id, user_id, start_date, end_date))
+        cursor.close()
```

Comment:
Should be unnecessary (here and below); when a `with` block exits, the context object is automatically closed